app/models/agents/website_agent.rb

```diff
@@ -40,6 +40,8 @@ class WebsiteAgent < Agent
             "description": { "path": "results.data[*].description" }
           }
 
+      If your JSON is compressed with `gzip`, set the `unzip` option to `true` to inflate it before extraction.
```

Comment:
It should. I didn't think that those other uses would come up, but there's no reason for me to assume that.